<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backie</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralMainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QWidget" name="pageMainScreen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="mainScreen">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="sidebarBase" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    width: 45px;
    height: 45px;
    border: none;
    border-radius: 5px;
	border-width: 0px;
	padding: 2px;	
}
QPushButton:hover, QPushButton:checked {
	background-color: #3A3A3A;
}
QWidget#sidebarBase {
    background-color: #282828;
}
</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <spacer name="verticalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>27</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="homeScreenB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/home_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/home_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/home_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>21</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="backupScreenB">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/backup_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/backup_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/backup_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>21</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="recoverScreenB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/recover_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/recover_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/recover_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>21</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="historyScreenB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/history_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/history_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/history_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>21</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string notr="true"/>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoRepeat">
                  <bool>false</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="autoRepeatDelay">
                  <number>300</number>
                 </property>
                 <property name="autoRepeatInterval">
                  <number>100</number>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>10</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="accountScreenB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/account_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/account_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/account_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>26</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>300</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QPushButton" name="settingsScreenB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>45</width>
                   <height>45</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="Resources.qrc">
                   <normaloff>:/resources/assets/icons/sideBar/notActive/settings_grey.svg</normaloff>
                   <normalon>:/resources/assets/icons/sideBar/active/settings_white.svg</normalon>:/resources/assets/icons/sideBar/notActive/settings_grey.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>21</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>true</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>27</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="secondaryStackedWidget">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <widget class="QWidget" name="page_9"/>
               <widget class="QWidget" name="page_10"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_8"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
